pkg/server/wire.go

```diff
@@ -196,11 +196,12 @@ var wireBasicSet = wire.NewSet(
 	dashboardimportservice.ProvideService,
 	wire.Bind(new(dashboardimport.Service), new(*dashboardimportservice.ImportDashboardService)),
 	plugindashboards.ProvideService,
+	sqlstore.ProvideService,
```

Comment:
I like that _unit_ tests no longer need a customer Wire graph. I wonder where this would potentially blow up.

Reply:
I changed it back how it was because of the tests failing https://github.com/grafana/grafana/pull/44762/commits/3fef6fc42b911b8e521fcfbf8a1ec1a5b785b084